opengrok-indexer/src/test/java/org/opengrok/indexer/configuration/SuggesterConfigTest.java

```diff
@@ -0,0 +1,22 @@
+package org.opengrok.indexer.configuration;
```

Comment:
Missing license?

Reply:
yep. Strange this was not caught by any check during build.